tests/Makefile

```diff
@@ -100,12 +100,12 @@ nfr-test: ## Run the NFR tests on a GCP VM
 	NFR=true bash scripts/run-tests-gcp-vm.sh
 
 .PHONY: start-longevity-test
-start-longevity-test: START_LONGEVITY=true
-start-longevity-test: nfr-test ## Start the longevity test to run for 4 days in GKE
+start-longevity-test:  ## Start the longevity test to run for 4 days in GKE
```

Comment:
it can just be changed to `export START_LONGEVITY=true`, I didn't know it was passed to a script